<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>перейти</string>
   </property>
  </widget>
  <widget class="QLabel" name="paragraph_label">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>20</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Оглавление</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>80</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1. ЭНЕРГОЭФФЕКТИВНОСТЬ ЗДАНИЙ И СООРУЖЕНИЙ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1_1">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>110</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1.1. Общая информация о здании</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1_1">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>перейти</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>140</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1.2. Расчетные условия функционирования здания</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1_2">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>перейти</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1_3">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>170</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1.3. Геометрические показатели здания</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1_3">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>перейти</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1_4">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>200</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1.4. Теплотехнические показатели</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1_4">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>перейти</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1_5">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>230</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1.5. Вспомогательные показатели</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1_8">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>320</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1.8. Пример расчета энергоэффективности  жилого здания</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1_6">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>перейти</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1_6">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>260</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1.6. Удельные характеристики здания</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1_7">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>перейти</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1_7">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>290</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1.7. Коэффициенты и комплексные показатели</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1_5">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>перейти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_1_8">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>перейти</string>
   </property>
  </widget>
  <widget class="QLabel" name="paragraph_label_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>420</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Что такое энергоэффективность и почему ее важно соблюдать?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>500</y>
     <width>471</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>Энергоэффективность жилого дома - это список показателей рационального и эффективного расхода энергии: экономное водоснабжение, отопление, вентиляцию и освещение. На энергоэффективность влияют и работа инженерного оборудования, и конструктивные особенности дома, и использованные стройматериалы</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>801</width>
     <height>691</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>776</width>
      <height>22686</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/1.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/2.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/3.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/4.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/5.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/6.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/7.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/8.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/9.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/10.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/11.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/12.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/13.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/14.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/15.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/16.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/17.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/18.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/19.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/20.jpg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/methodic/21.jpg</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
